packages/forms/backend/sirius-components-collaborative-forms/src/main/resources/schema/form.graphqls

```diff
@@ -93,6 +93,7 @@ type Checkbox implements Widget {
 
 type CheckboxStyle {
   color: String
+  labelPlacement: String!
```

Comment:
Do we really want to set this attribute mandatory? In CheckboxWidget.tsx, the value 'end' has been hard-coded if no value has been set.
So unless I'm missing something here, I don't think this attribute should be mandatory.

Reply:
Furthermore, in the form.ecore, `labelPlacement` has been defined as an EEnum.
So I don't think this should be a `String` here.
Check about `GroupDisplayMode` to see how it has been done for other EEnum as an example.

Reply:
In the form.ecore, I made it mandatory but I can change that.
What is the radio button behavior when there is no value ?

Reply:
Ok so indeed it should stay mandatory, because null value does not have meaning

Reply:
Argh, we are not consistent here. Ok it's because you use the values from MUI, as we did for the flexDirection. Ok I understand, you can keep the string values if you want.